dxr/plugins/python/indexers.py

```diff
@@ -260,10 +260,11 @@ def analyze_tokens(self):
                 # names, we start the queue for the current parenthesis level
                 # at the byte offset for the keyword token, but only start
                 # pushing character offsets once we're past the keyword.
-                paren_stack.setdefault(paren_level, (utf8_start, []))
+                queue_start, node_queue = paren_stack.setdefault(paren_level, (utf8_start, []))
+                node_start_table.setdefault(queue_start, node_queue)
```

Comment:
Yep.  queue_start is the index of the starting point some chain of attributes, e.g. `foo.bar.baz`, and the node queue needs to be a list of the starting locations of each of the attributes in that chain.  If we've already started capturing the indexes of such a chain, we want to keep on with the same list.